intake/source/tests/plugin_searchpath/driver_with_entrypoints/__init__.py

```diff
@@ -0,0 +1,2 @@
+class SomeTestDriver:
+    ...
```

Comment:
Does this get loaded? Should be `pass`?

Reply:
It does get loaded. Ellipsis is a valid substitute for `pass` in py3.

Reply:
Still learning....

Reply:
Me too! :-D